lib/WorldState.js

```diff
@@ -79,11 +78,12 @@ class WorldState {
    * @param {string} json The worldstate JSON string
    * @param {Object} [deps] The options object
    */
-  constructor(json, deps = defaultDeps) {
+  constructor(json, lang = 'en_US', deps = defaultDeps) {
```

Comment:
This enhancement (all translation) implies a full rework of return types.
There is multiple choices to achieve that. In worldState, we can have localization object for all structure. For example `worldState.alerts` become `worldState.fr_FR.alerts`. It's easy to switch/use a language and all languages are available in a single object.
We can also modify each submodel (Alert, ...) to add translations on it but it's less "fluent" to use (ex: `worldState.alerts[0].translations.fr_FR...`) and more difficult to represent in classes. Which goal would you prefer ?